package.json

```diff
@@ -120,6 +120,7 @@
     "react-test-renderer": "^18.2.0",
     "source-map-explorer": "^2.5.2",
     "tailwindcss": "^3.2.7",
+    "tailwindcss-aria-attributes": "^1.3.0",
```

Comment:
@abbasali1994, `aria-*` states are already included in `3.2.7`. I think there's no need for plugin for this - As other classes like `aria-disabled:*` or `aria-selected:*` is working

Reply:
aria-invalid isnt working as part of class def without the use of dependency (gives undefined error), but when defined seperately it works for tailwindcss, so modified index.css to remove the additional dependency